src/sass/_recommendations_page.scss

```diff
@@ -0,0 +1,72 @@
+.card-list {
+  padding-left: 0.0625rem;
+  padding-bottom: 0.125;
+  @include media-breakpoint-down(xl) {
+    overflow-x: scroll;
+    overflow-y: hidden;
+  }
+}
+.recommendations-heading {
+  overflow-wrap: break-word;
+}
+
+.recommendations-container {
+  padding: 0 1rem;
+  margin: 0 0 1.875rem 0;
+  @include media-breakpoint-up(lg) {
+    max-width: $max-width-sm + 5 * $grid-gutter-width !important;
+  }
+
+  @include media-breakpoint-up(xl) {
+    max-width: $max-width-md + $grid-gutter-width !important;
+  }
+
+  @include media-breakpoint-up(xxl) {
+    max-width: $max-width-lg + $grid-gutter-width !important;
+  }
+}
+
+.recommendation-card {
+  .pgn__hyperlink {
+    display: block;
+  }
+
+  .pgn__card {
+    width: 281px;
+    height: 332px;
+    margin: 0 !important;
+  }
+
+  .pgn__card-image-cap {
+    height: 6.5rem;
+  }
+
+  .pgn__card-header-title-md {
+    font-family: 'Inter';
```

Comment:
No need to add font-family and font-style, I think it is already following this spec because MFE uses paragon branding